react.js

```diff
@@ -1,8 +1,9 @@
-'use strict'
+"use strict";
 
 module.exports = {
-  extends: [
-    require.resolve('tslint-react'),
-    require.resolve('./'),
-  ]
-}
+  extends: [require.resolve("tslint-react"), require.resolve("./")],
```

Comment:
@yowainwright noice.

@kylecorbelli just added a few more to face-manager as well, fyi.